code/modules/atmospherics/gasmixtures/gas_mixture.dm

```diff
@@ -221,7 +246,7 @@ GLOBAL_LIST_INIT(gaslist_cache, init_gaslist_cache())
 	removed.temperature = temperature
 	for(var/id in cached_gases)
 		ADD_GAS(id, removed.gases)
-		removed_gases[id][MOLES] = QUANTIZE(cached_gases[id][MOLES] * ratio)
+		removed_gases[id][MOLES] = QUANTIZE_TO_MOVE(cached_gases[id][MOLES] * ratio)

```

Comment:
See above

Reply:
yes it now do something wrong. I think garbage delete negative micro moles but it pass it somewhere.